microsoft-edge/webview2/reference/HoloLens2/webview2-unity-plugin.md

```diff
@@ -0,0 +1,316 @@
+---
+title: WebView plugin for Unity API Reference (Preview)
+description: API Reference for Microsoft Mixed Reality WebView plugin for Unity.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 05/08/2023
+---
+# Mixed Reality WebView Plugin for Unity API Reference (Preview)

```

Comment:
```suggestion
# API Reference for Mixed Reality WebView Plugin (Preview)
```
check casing of "API Reference" at live: it's init-cap R:
https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/javascript/

Reply:
done locally, can close